uaaextras/templates/invite.html

```diff
@@ -20,4 +20,5 @@
         <input name=_csrf_token type=hidden value="{{ csrf_token() }}">
      </form>
 </div>
+<p><a href="">Learn more about inviting teammates</a></p>
```

Comment:
Presumably this href should point to https://cloud.gov/docs/apps/managing-teammates/ ?

Also, this is used in two cases, one of which is specifically about self-inviting from an unauthenticated context... If we only reference managing teammates, then it may confuse people.

How about `Learn more about [who can use cloud.gov](https://cloud.gov/overview/overview/who-can-use-cloudgov/) and [inviting teammates](https://cloud.gov/docs/apps/managing-teammates/)`